quint/src/runtime/trace.ts

```diff
@@ -218,16 +226,26 @@ class TraceRecorderImpl implements TraceRecorder {
     // https://github.com/informalsystems/quint/issues/747
     if (verbosity.hasUserOpTracking(this.verbosityLevel)) {
       const newFrame = { app: app, args: [], result: none(), subframes: [] }
-      if (this.frameStack.length > 0) {
+      if (this.frameStack.length == 0) {
+        // this should not happen, as there is always bottomFrame, but let's handle it
+        this.frameStack = [newFrame]
```

Comment:
If its an erroneous state as far as we know, should it not raise an error?

Reply:
in principle, yes, but the trace output is a secondary feature, so I would prefer not to fail on that.